contracts/src/BorrowerOperations.sol

```diff
@@ -1350,26 +1347,14 @@ contract BorrowerOperations is LiquityBase, AddRemoveManagers, IBorrowerOperatio
         }
     }
 
-    function _requireNotBelowCriticalThreshold(uint256 _price) internal view {
-        if (_checkBelowCriticalThreshold(_price, CCR)) {
-            revert BelowCriticalThreshold();
```

Comment:
This makes `error BelowCriticalThreshold()` unused, I believe.

Reply:
Oh, right, I removed `BorrowingNotPermittedBelowCT` but forgot this one. Thanks for fixing it!